src/simulation/dynamics/spinningBodies/spinningBodiesTwoDOF/spinningBodyTwoDOFStateEffector.cpp

```diff
@@ -37,6 +37,20 @@ SpinningBodyTwoDOFStateEffector::SpinningBodyTwoDOFStateEffector()
     this->IS2PntSc2_S2.setIdentity();
     this->dcm_S10B.setIdentity();
     this->dcm_S20S1.setIdentity();
+    this->ATheta.setZero();
+    this->BTheta.setZero();
+    this->rTilde_Sc1B_B.setIdentity();
```

Comment:
Can you make lines 42-46 and 52-53 use ``setZero()``?

Reply:
Done